L1Trigger/TrackFindingTracklet/test/L1TrackNtupleMaker_cfg.py

```diff
@@ -111,22 +113,24 @@ def getTxtFile(txtFileName):
     NHELIXPAR = 4
     L1TRK_NAME  = "TTTracksFromTrackletEmulation"
     L1TRK_LABEL = "Level1TTTracks"
+    L1TRUTH_NAME = "TTTrackAssociatorFromPixelDigis"
 
 # LEGACY ALGORITHM (EXPERTS ONLY): TMTT  
 elif (L1TRKALGO == 'TMTT'):
     print "\n WARNING - this is not a recommended algorithm! Please use HYBRID (HYBRID_DISPLACED)! \n"
     process.load("L1Trigger.TrackFindingTMTT.TMTrackProducer_Ultimate_cff")
-    L1TRK_PROC  =  process.TMTrackProducer
+    L1TRK_PROC  =  process.TMTrackProducer    
```

Comment:
remove unnecessary whitespace at end of line

Reply:
done